### PR TITLE
#494 Resolve issue with not showing messages of loglevel INFO

### LINKED ISSUE #494: [BUG] Logger won't display level INFO and won't use uvicorn handler when running locally
## Brief description:
When using a logger with level INFO, logs won't get displayed by the uvicorn handler. 
When using a higher level, a log will get displayed but with no applied formatting at all. 
Additionally, I think it will be a good idea to have the handler show the name of the current logger.

E.g. `[INFO][auth]: Running authentication locally`

### PR DESCRIPTION
**Pull request by IvanObreshkov**
Fixed the issue with not showing messages of log-level INFO and not formatting the ones from level WARNING and above by overwriting uvicorn's default logging config.

resolves #494

### CODE REVIEW

File: packages/py-api/py_api/utilities/logging.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
-from os import getenv
+import logging
```
Comment (asynchroza) [inline packages/py-api/py_api/utilities/logging.py:1]: Drop this

File: packages/py-api/py_api/utilities/logging.py [hunk 2]
```diff
@@ -32,10 +35,36 @@
...
         IS_OFFLINE,
     ) else LOGGING_CONFIG
 
     return log_config
+
+
+fmt = Formatter(fmt="[%(levelprefix)s] [%(name)s]: %(message)s")
+
+
+def formatMessage(self: ColourizedFormatter, record: logging.LogRecord) -> str:
```
Comment (asynchroza) [inline packages/py-api/py_api/utilities/logging.py:58]: from logging import LogRecord

File: packages/py-api/py_api/utilities/logging.py [hunk 3]
```diff
@@ -32,10 +35,36 @@
     },
 }
 
+# Overwrite uvicorn's default logging config in order to be able to log INFO level messages
```
Comment (asynchroza) [inline packages/py-api/py_api/utilities/logging.py:38]: Let's rephrase this to:

Overwrites uvicorn's default logging config in order to enable INFO level when using our own logger handlers

**Comment by IvanObreshkov** [comment]
@asynchroza we should be good to go

**Comment by asynchroza** [comment]
> @asynchroza we should be good to go

FYI @IvanObreshkov you can rerequest a review by clicking the icon next to my handle:
image

**Review by asynchroza — APPROVED** [review]
Amazing job on this one :raketa:. I feel like this ticket turned out to be more beneficial than expected :kriskobeats:

**Comment by IvanObreshkov** [comment]
> Amazing job on this one :raketa:. I feel like this ticket turned out to be more beneficial than expected :kriskobeats:

Thank you mate!

**Comment by asynchroza** [comment]
@IvanObreshkov one last note - can you update the description of the PR since it doesn't really reflect what you actually did.
